contrib/test_jenkins.sh

```diff
@@ -574,6 +574,70 @@ run_ucx_perftest() {
 	fi
 }
 
+start_daemon() {
```

Comment:
CI should be similar to how ucx_perftest MAD is run - using different steps in Azure pipelines jobs, including a step that cleans up the deamon in case of failure
we can move CI scripts from this PR to next PR